projects/1.20/assets/ex-machinis/exmachinis/lang/zh_cn.json

```
@@ -0,0 +1,64 @@
+{
+  "advancement.exmachinis.diamond_upgrade": "一次处理更多，更快！",
+  "advancement.exmachinis.diamond_upgrade.desc": "合成一个钻石升级",
+  "advancement.exmachinis.flux_compactor": "把它装回原样",
+  "advancement.exmachinis.flux_compactor.desc": "在仓库里放满了讨厌的方块？合成一个通量压缩机",
+  "advancement.exmachinis.flux_hammer": "砸碎它！",
+  "advancement.exmachinis.flux_hammer.desc": "不需要再锤击沙子了！合成一个通量锤",
+  "advancement.exmachinis.flux_sieve": "筛矿⋯⋯但是通量筛子！",
+  "advancement.exmachinis.flux_sieve.desc": "疲于一直右键点击？合成一个通量筛子吧",
+  "advancement.exmachinis.gold_upgrade": "一次处理更多！",
+  "advancement.exmachinis.gold_upgrade.desc": "制作一个金升级",
+  "advancement.exmachinis.netherite_upgrade": "这就结束了？",
+  "advancement.exmachinis.netherite_upgrade.desc": "“不，这只是下界合金”。制作一个下界合金升级",
+  "advancement.exmachinis.root": "Ex Machinis",
+  "advancement.exmachinis.root.desc": "源于机器的自动化",
+  "block.exmachinis.flux_compactor": "通量压缩机",
+  "block.exmachinis.flux_hammer": "通量锤",
+  "block.exmachinis.flux_sieve": "通量筛子",
+  "block.exmachinis.item_buffer": "物品缓存器",
+  "exmachinis.int.jei.category.compacting": "通量压缩机",
+  "exmachinis.int.jei.category.flux_compactor": "通量压缩机将从内部库存中取出物品，\n并输出至下方容器中。\n\n添加升级可以增加可以一次处理物品的数量。\n\n可以提供红石信号禁用。\n\n输出面可以通过手持Ex Nihilo的任意锤Shift+右击调整。\n",
+  "exmachinis.int.jei.category.flux_hammer": "通量锤将从上方容器中取出物品，\n并输出至前方容器中。\n\n添加升级可以增加可以一次处理物品的数量。\n\n可以提供红石信号禁用。\n\n输出面可以通过手持Ex Nihilo的任意锤Shift+右击调整。\n",
+  "exmachinis.int.jei.category.flux_sieve": "通量筛子将从内部库存中取出物品，\n并输出至下方容器中。\n\n添加升级可以增加可以一次处理物品的数量。\n\n筛网可以附魔时运来增加掉落几率。\n\n可以提供红石信号禁用。\n\n输出面可以通过手持Ex Nihilo的任意锤Shift+右击调整。\n",
+  "exmachinis.int.jei.category.item_buffer": "物品缓存器将从内部库存中获取项目，并输出至下方容器中。\n\n可以添加升级以增加一次处理的项目数量。\n\n比较器升级后可以增加在特定方向产生红石信号。\n你可以用它来关闭其他机器。\n\n方块可以通过红石信号禁用。\n\n输出面可以通过手持Ex Nihilo的任意锤Shift+右击调整。\n",
+  "gui.exmachinis.inventoryFull": "库存已满",

```

```suggestion
  "gui.exmachinis.inventoryFull": "内部存储已满",
```
inventory：玩家的物品栏；容器；（容器/机器的）内部存储空间